packages/core/src/modules/connections/repository/ConnectionRecord.ts

```diff
@@ -38,6 +39,10 @@ export type DefaultConnectionTags = {
   theirDid?: string
   outOfBandId?: string
   invitationDid?: string
+  connectionTypes?: Array<ConnectionType | string>
```

Comment:
While I have added this here, if we are doing away with adding connection types to tags, do we need this anymore?

Reply:
We still need to add them to the tags, it's just based on the record value now

Reply:
I have changed the methods in the connection service to add and remove directly from the record now, I suppose I should update the `getTags` method in the record to also return the value for `this.connectionTypes`?